src/pages/junghyowon/Login/JungLogin.scss

```
@@ -0,0 +1,119 @@
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
+
+.container-wrap {
+  width: 350px;
+  margin: 50px auto 10px auto;
```

```suggestion
  margin: 50px auto 10px ;
```
더 간단하게 표현할 수 있을거 같아요!